src/services/chris_integration.tsx

```diff
@@ -229,6 +229,17 @@ class ChrisIntegration {
     return feeds.getItems()?.[0]?.data?.id;
   }
 
+  /**
+   * Returns true if the plugin with the given id is in a terminated state (SUCCESS, ERROR, or CANCELLED)
+   * @param {number} id
+   */
+  static async checkIfPluginTerminated(id: number): Promise<boolean> {
+    const client: any = ChrisAPIClient.getClient();
```

Comment:
Please, avoid using `any` when type is known.

```suggestion
    const client: Client = ChrisAPIClient.getClient();
```

Reply:
Done https://github.com/darwinai/covidnet_ui/pull/35/commits/adf1b2ad037e5790dae81368b46ee042a4e4d2a7